### ISSUE TITLE
Does not build with original Spark ML library

### ISSUE THREAD

**Issue by shry15harsh** [RedisLabs/spark-redis-ml#4]
I get the error of missing getImpurityStats() method in Spark ML package. I checked the Dockerfile because in Docker it was running fine, then I found out that modified Spark distribution is used there in which this method has been added.
The pull request and the corresponding issue is still in progress on Spark JIRA. Nowhere in the steps, it is mentioned. Also, RedisLabs has written a lot about Spark-Redis integration on its website and Databricks gave presentation and tutorial. So I am wondering why nobody discussed it or is it just me for whom it is breaking?

**Comment by shaynativ** [comment]
Good point. On earlier versions the impurity stats weren't required. I'll try to speed up the pull request on Spark and meanwhile add this point the documentation.

**Comment by shry15harsh** [comment]
Thanks. It would be really awesome if it gets in Spark. It would be really a scalable and fast model serving solution.
